addons/binding/org.openhab.binding.gpstracker/src/main/java/org/openhab/binding/gpstracker/internal/BindingComponent.java

```diff
@@ -136,8 +136,11 @@ protected void activate(ComponentContext componentContext) {
         updatePrimaryLocationFromSystemConfig();
 
         logger.debug("Initializing discovery service");
-        this.discoveryService = new TrackerDiscoveryService();
-        this.serviceRegistration = this.bundleContext.registerService(DiscoveryService.class.getName(), this.discoveryService, new Hashtable<>());
+        TrackerDiscoveryService discoveryService = new TrackerDiscoveryService();
+        this.serviceRegistration = this.bundleContext.registerService(DiscoveryService.class.getName(),
+                discoveryService, new Hashtable<>());
+
+        this.translationUtil = new TranslationUtil(getBundleContext());
```

Comment:
@kaikreuzer should we allow this? In the light of eclipse/smarthome#5182 we want to get rid of the OSGi ServiceTrackers in the `BaseThingHandler` so this has be avoided, too? Hasn't it?

Reply:
@cweitkamp If I see it right, this is in the HandlerFactory (I already asked @gbicskei to use that name again, instead of "BindingComponent"), so this is the class which is a DS service and thus has the bundle context and can use other services. So I would say this is alright here as long as the bundleContext is not reached to the handler instances.

Reply:
Still pretty generic. The skeleton would create it as `GPSTrackerHandlerFactory`.

Reply:
Done